working/0110-incremental-sound-nnbd/roadmap.md

```diff
@@ -95,14 +95,14 @@ these checks should be provably redundant and do not need to be inserted.
 
 Concretely, opting in to weak null checking will proceed as follows:
 - The programmer opts into the weak null checks by marking their package as
-  opted in.
+  opted in. They can opt out of weak null checks on a per library basis.
```

Comment:
I'd expect this to also opt them out of the syntax. It's opting out of the entire feature, not just static checking.

Reply:
Agreed. That is my intent. I'll update the PR to indicate as much.